public/hwcdefs.h

```diff
@@ -63,5 +63,12 @@ enum class DisplayType : int32_t {
   kHeadless = 3
 };
 
+enum DisplayPowerMode {
+  kOff = 0,
```

Comment:
Also, can u run git diff | clang-format-diff-3.5 -p 1 -style=file to fix some indentation issues.